src/AutoMerge/cron.jl

```diff
@@ -384,7 +394,13 @@ function cron_or_api_build(
         return nothing
     end
 
-    if !pr_has_no_blocking_comments(api, registry, pr; auth=auth)
+    blocked = pr_has_blocking_comments(api, registry, pr; auth=auth) && !has_label(pr.labels, OVERRIDE_BLOCKS_LABEL)
+    if blocked
+        if !read_only
+            # add `BLOCKED_LABEL` to communicate to users
+            # that the PR is blocked from automerging
+            GitHub.add_labels(api, registry.full_name, pr_number, [BLOCKED_LABEL]; auth=auth)
```

Comment:
Do I understand correctly this unconditionally adds the label even if it's there already?  What I'm seeing is that [PRs with this label](https://github.com/JuliaRegistries/General/pulls?page=2&q=sort%3Aupdated-desc+is%3Apr+is%3Aopen+label%3A%22AutoMerge%3A+last+run+blocked+by+comment%22) are "updated" every time automerge runs, even if the label is already there, with the result that these PRs will never be automatically closed as stale, as they're continuously updated every 10 minutes or so.  Also, this makes looking at the list of [recently updated PRs](https://github.com/JuliaRegistries/General/pulls?q=sort%3Aupdated-desc+is%3Apr+is%3Aopen) hard, because these PRs are constantly bumped (and there are dozens of them, especially since they can't be automatically closed) and so there are dozens of them, covering other PRs that had been last updated mere minutes earlier than last automerge run.